edm4eic.yaml

```
@@ -283,6 +283,7 @@ datatypes:
     VectorMembers:
       - edm4eic::CherenkovParticleIDHypothesis hypotheses         // Evaluated PDG hypotheses
       - edm4hep::Vector2f                      thetaPhiPhotons    // estimated (theta,phi) for each Cherenkov photon
+      - float                                  incidentAngle      // incident photon angle wrt sensor normal
```

We should clarify this is MC truth
```suggestion
      - float                                  incidentAngle      // incident MC-truth photon angle wrt sensor normal
```

---

can we also mention "MC-truth" in the descriptions for `refractiveIndex` and `photonEnergy` too, just so it's clear these come from MC truth, and not reconstruction?